src/jit/rationalize.h

```diff
@@ -39,22 +39,22 @@ class Rationalizer : public Phase
     void RewriteSIMDOperand(LIR::Use& use, bool keepBlk);
 
     // Intrinsic related transformations
-    void RewriteNodeAsCall(GenTree**             use,
-                           ArrayStack<GenTree*>& parents,
-                           CORINFO_METHOD_HANDLE callHnd,
+    void RewriteNodeAsCall(GenTree**               use,
+                           Compiler::GenTreeStack& parents,
```

Comment:
Why the change?

Reply:
I'd keep `ArrayStack<GenTree*>`. It's not some convoluted template instantiation that warrants the use of a typedef and the `GenTreeStack` typedef is used only in 3 places so it's best not to add more (and remove it at some point).